src/main/java/mate/academy/spring/service/CustomUserDetailsService.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+    @Autowired
```

Comment:
Remove the unnecessary @Autowired annotation since you are already using constructor injection.